redfish-core/lib/log_services.hpp

```diff
@@ -1475,6 +1528,219 @@ inline void requestRoutesJournalEventLogEntry(App& app)
+                        }
+                    }
+                    else if (propertyMap.first == "Severity")
+                    {
+                        severity =
+                            std::get_if<std::string>(&propertyMap.second);
+                        if (severity == nullptr)
+                        {
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                    }
+                    else if (propertyMap.first == "Message")
+                    {
+                        message = std::get_if<std::string>(&propertyMap.second);
+                        if (message == nullptr)
+                        {
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                    }
+                    else if (propertyMap.first == "Resolved")
+                    {
+                        bool* resolveptr =
+                            std::get_if<bool>(&propertyMap.second);
+                        if (resolveptr == nullptr)
+                        {
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        resolved = *resolveptr;
+                    }
+                }
+            }
+            else if (interfaceMap.first ==
+                     "xyz.openbmc_project.Common.FilePath")
+            {
+                for (auto& propertyMap : interfaceMap.second)
+                {
+                    if (propertyMap.first == "Path")
+                    {
+                        filePath =
+                            std::get_if<std::string>(&propertyMap.second);
+                    }
+                }
+            }
+            else if (interfaceMap.first == "org.open_power.Logging.PEL.Entry")
+            {
+                for (auto& propertyMap : interfaceMap.second)
+                {
+                    if (propertyMap.first == "Hidden")
+                    {
+                        bool* hiddenPropPtr =
+                            std::get_if<bool>(&propertyMap.second);
+                        if (hiddenPropPtr == nullptr)
+                        {
+                            BMCWEB_LOG_ERROR << "Unable to retrieve hidden "
+                                                "property";
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        hiddenPropVal = *hiddenPropPtr;
+                        break;
+                    }
+                }
+            }
+        }
+        // Object path without the
+        // xyz.openbmc_project.Logging.Entry interface, ignore
+        // and continue.
+        if (id == nullptr || message == nullptr || severity == nullptr)
```

Comment:
Should you add hidden here? If we don't have it continue?

Reply:
Done